src/langchain_google_alloydb_pg/__init__.py

```diff
@@ -12,8 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from langchain_google_alloydb_pg import version as package_version
```

Comment:
Actually let's just use "from .version import __version__"

Reply:
Should I just do `from langchain_google_alloydb_pg.version import version` to make it consistent with the others instead?